domains/korsinemi.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Korsinemi",
+        "email": "",
+        "twitter": "@Korsinemi"
+    },
+    "record": {
+        "TXT": ["_discord.korsinemi.is-a.dev", "dh=288b69b6df6598410e14652112ee7609f689c8fb"],
```

Comment:
btw if you want to link it to discord, that's not the correct way, you should create another file `_discord.korsinemi.json` and put the TXT record in there